config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml

```diff
@@ -1,29 +1,30 @@
 periodics:
-- name: periodic-sync-enhancements-github-project-1-27
-  interval: 6h
-  cluster: k8s-infra-prow-build-trusted
-  decorate: true
-  annotations:
-    testgrid-create-test-group: "true"
-    testgrid-dashboards: sig-release-release-team-periodics
-    description: Periodically syncs the opted-in k/enhancements issues on the Enhancements Tracking GitHub Beta Project Board
-  extra_refs:
-  - org: kubernetes
-    repo: sig-release
-    base_ref: master
-  spec:
-    containers:
-    - image: gcr.io/k8s-staging-infra-tools/k8s-infra:latest
-      command:
-      - ./release-team/hack/sync-enhancements-github-project-beta.sh
-      env:
-        - name: GITHUB_PROJECT_BETA_NUMBER
-          value: "117"
-        - name: GITHUB_TOKEN
-          valueFrom:
-            secretKeyRef:
-              name: k8s-release-enhancements-triage-github-token
-              key: token
+# Re-enable once v1.28 project board is created (and update the project board number below)
```

Comment:
Suggestion: you can make a much smaller diff with just changing the interval field to an impossible cron date

Reply:
that means things like dashboards won't go anywhere, and the revert PR is a quicker review

Reply:
done